recipes/torchvision/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: torchvision
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/pytorch/vision.git
```

Comment:
Please build from tarballs
https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos